bootcamp/tutorials/integration/function_calling.ipynb

```diff
@@ -0,0 +1,450 @@
+{
```

Comment:
<p>By doing so, our agent can then search through the corpus and pick semantically related information for question answering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>